gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -24,6 +24,14 @@
  */
 
 #include <string.h>
+#if defined(__aarch64__) || defined(__arm__)
+#include <sys/auxv.h>
+#if defined(__ANDROID__)
+#include <asm/hwcap.h>
+#else /* __ANDROID __ */
+#include <asm-arm/hwcap.h>
+#endif /* __android __ */
+#endif /* __arch64__ __arm__ */
```

Comment:
IMO, it's better to create an additional header file like this [framework](https://ncnn.docsforge.com/master/api/ncnn/cpu_support_arm_neon/) in another PR,

Reply:
Yes, the accelerator related stuff will move to different files.